jetty-client/src/main/java/org/eclipse/jetty/client/HttpReceiver.java

```diff
@@ -755,6 +752,7 @@ private Decoder(HttpExchange exchange, ContentDecoder decoder)
         {
             this.exchange = exchange;
             this.decoder = Objects.requireNonNull(decoder);
+            decoder.beforeDecoding(exchange);
```

Comment:
No `try / catch` protection around this call?

Reply:
We don't try/catch for the decoding either, so no.